Crafttweaker_Annotation_Processors/src/main/java/com/blamejared/crafttweaker_annotation_processors/processors/document/conversion/converter/comment/event/EventDataConverter.java

```diff
@@ -0,0 +1,114 @@
+package com.blamejared.crafttweaker_annotation_processors.processors.document.conversion.converter.comment.event;
+
+import com.blamejared.crafttweaker_annotation_processors.processors.document.conversion.converter.comment.documentation_parameter.ParameterInfo;
+import com.blamejared.crafttweaker_annotation_processors.processors.document.conversion.converter.comment.documentation_parameter.ParameterInformationList;
+import com.blamejared.crafttweaker_annotation_processors.processors.document.conversion.converter.comment.documentation_parameter.ParameterReader;
+import com.blamejared.crafttweaker_annotation_processors.processors.document.page.comment.DocumentationComment;
+import com.blamejared.crafttweaker_annotation_processors.processors.document.page.member.header.examples.ExampleData;
+
+import javax.lang.model.element.Element;
+
+public class EventDataConverter {
+
+    private static final String DOC_EVENT = "docEvent";
+    private static final String CANCELED = "canceled";
+    private static final String NOT_CANCELED = "notCanceled";
+    private static final String ALLOW = "allow";
+    private static final String DEFAULT = "default";
+    private static final String DENY = "deny";
+    private static final String NOT_CANCELED_INFO = "The event is not cancelable.";
+    private static final String NO_RESULT_INFO = "The event does not have a result.";
+    private static final String CANCELED_INFO = "The class is cancelable.";
```

Comment:
Shouldn't this instead say `The event is is cancelable.`, since everything else is talking about events?
```suggestion
    private static final String CANCELED_INFO = "The event is cancelable.";
```